game/rooms/room06/esc/worker.esc

```diff
@@ -1,5 +1,5 @@
 :look
-say player "He's looking a bit odd."
+say player "He's wearing a badge that says "Careers Officer"."
```

Comment:
Do the quotation marks around "Careers Officer" work all right? Do they need escaping?

Reply:
They don't need escaping.
![image](https://user-images.githubusercontent.com/5151242/169043739-820bd1de-d2f1-46cb-8fa1-4cea851114b4.png)

Reply:
#unexpectedparsingwin